src/main/msp/msp.c

```diff
@@ -1605,13 +1605,15 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, blackboxGetRateDenom());
         sbufWriteU16(dst, blackboxGetPRatio());
         sbufWriteU8(dst, blackboxConfig()->sample_rate);
+        sbufWriteU32(dst, blackboxConfig()->fields_disabled_mask);
```

Comment:
In order to make debugging easier, can you please add comments indicating the MSP version that this was added, like in https://github.com/betaflight/betaflight/pull/10054/files#diff-eefd576c86ccfa8b6cdf98081806a7a8L3096.

Also, please keep in mind that this will only be going into 4.4, so this pull request will have to stay open for months before it is merged, and will probably have to be rebased a number of times before this.

Reply:
Ok, no problem for me. I will add it as msp 1.44 for now, will be changed in future